### PR TITLE
Update Dishonored 2 AutoSplitter

### PR DESCRIPTION
**Pull request by soppelsad**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -5370,11 +5370,11 @@
...
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/HuthTV/asl/master/dishonored2.asl</URL>
+            <URL>https://raw.githubusercontent.com/soppelsad/asl/master/dishonored2.asl</URL>
         </URLs>
         <Type>Script</Type>
-        <Description>Load Removal and Auto Start is available. (By DrTChops, Som1Lse and HuthTV)</Description>
-        <Website>https://github.com/HuthTV/asl/blob/master/README.md</Website>
+        <Description>Load Removal and Auto Start is available. (By DrTChops, Som1Lse, HuthTV and Elvyndir)</Description>
+        <Website>https://raw.githubusercontent.com/soppelsad/asl/master/README.md</Website>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:5377]: This shouldn't be a raw link because it opens in the browser as a text file

**Review by wooferzfg — COMMENTED** [review]
Did you get permission from the authors to replace their auto splitter?

File: LiveSplit.AutoSplitters.xml (hunk 1 above)
Comment (soppelsad) [reply LiveSplit.AutoSplitters.xml]: My bad, fixed it

**Comment by soppelsad** [comment]
> Did you get permission from the authors to replace their auto splitter?

It wasn't maintenained anymore, README stated : 

> These scripts are no longer maintained. If something needs to be updated, fork the repo or copy the script somewhere else, host it yourself, and point livesplit to the new version.

**Pull request merged**